assets/chef-recipes/cookbooks/mariadb/recipes/uninstall.rb

```diff
@@ -1,19 +1,49 @@
+service 'mariadb' do
```

Comment:
Please check enterprise_install recipe for information on how these services can be named.